src/main/kotlin/net/axay/hglaborlobby/eventmanager/joinserver/JoinPlayerReset.kt

```diff
@@ -10,6 +10,9 @@ object JoinPlayerReset {
 
         player.feedSaturate()
         player.heal()
+        for (potionEffect in player.activePotionEffects) {
```

Comment:
Are you sure, that this does not throw a `ConcurrentModificationException`?

Reply:
Yes, I've used this a lot